src/main/flight/mixer.c

```diff
@@ -590,11 +590,11 @@ void mixTable(uint8_t vbatPidCompensation)
 
     // Calculate and Limit the PIDsum
     const float scaledAxisPidRoll =
-        constrainf((axisPID_P[FD_ROLL] + axisPID_I[FD_ROLL] + axisPID_D[FD_ROLL]) / PID_MIXER_SCALING, -pidSumLimit, pidSumLimit);
```

Comment:
I think the existing implementation is correct, since `pidSumLimit` is scaled down to `0.1` ... `1.0` in https://github.com/martinbudden/cleanflight/blob/fc983008858d660ae04906917844e106ac63b692/src/main/flight/mixer.c#L384.

However, looking at the code, we could eliminate the scaling and use `pidProfile->pidSumLimit` directly if we applied the constraint before the `/ PID_MIXER_SCALING`.